src/receive_imf.rs

```diff
@@ -361,6 +361,10 @@ pub(crate) async fn receive_imf_inner(
         .handle_reports(context, from_id, sent_timestamp, &mime_parser.parts)
         .await;
 
+    if mime_parser.is_bot {
```

Comment:
Setting it back implies, that it was wrongly set initially because a out-of-office notice should never set the bot flag anyway. I thought it would be nice to do one less db-access here, but in the end it probably doesn't matter so we can add it.

Reply:
I think resetting it makes sense because it's also a way to do a recovery if it was set by a mistake